test/fluree/db/ledger/query/analytical_test.clj

```diff
@@ -183,7 +183,144 @@
       (is (= [{"handle"  "jakethesnake", :_id 351843720888324
                "favNums" [7 42], "user" {:_id 87960930223081}}]
              res)
-          (str "Unexpected query result: " (pr-str res))))))
+          (str "Unexpected query result: " (pr-str res)))))
+  (testing "query with one var binding should resolve correctly"
+    (let [query  {:where  [["?p" "person/handle" "?handle"]
+                           ["?p" "person/favNums" "?favNums"]
+                           ["?p" "person/user" "?username"]]
+                  :select {"?p" ["handle" "favNums" "user"]}
+                  :vars   {"?handle" ["jakethesnake"]}}
+          ledger (test/rand-ledger test/ledger-chat
+                                   {:http/schema ["chat.edn" "chat-preds.edn"]})
+          {:keys [block]} (test/transact-data ledger "chat.edn")
+          db     (fdb/db (:conn test/system) ledger {:syncTo block})
+          res    (<?? (fdb/query-async db query))]
+      (is (= [{"handle"  "jakethesnake", :_id 351843720888324
+               "favNums" [7 42], "user" {:_id 87960930223081}}]
+             res)
+          (str "Unexpected query result: " (pr-str res)))))
+  (comment
+   ;; For some reason these multiple binding vars aren't working the way I'd
+   ;; expect w/ these data and/or queries. They only return one result where I'd
+   ;; expect more than one to match. Might be a bug.
+   ;; - WSM 2023-06-06
```

Comment:
I think it makes sense to add a low priority bug in the icebox for this just so we don't forget about it if we ever need to revisit this.